spring-boot-project/spring-boot-docs/src/main/asciidoc/using-spring-boot.adoc

```diff
@@ -185,8 +185,8 @@ section if you want to use it:
 	</build>
 ----
 
-NOTE: If you use the Spring Boot starter parent pom, you only need to add the plugin,
-there is no need for to configure it unless you want to change the settings defined in
+NOTE: If you use the Spring Boot starter parent pom, you need to add only the plugin.
+There is no need for to configure it unless you want to change the settings defined in
```

Comment:
There's an extra "for" here. Not your doing, but thought I'd mention it as I noticed it during this review.

Reply:
Yup. Thanks.